lib/puppet/util/rundeck_acl.rb

```diff
@@ -1,314 +1,235 @@
-module Puppet::Util::RundeckACL
-  
-  class RundeckValidator
-
-    def raise_err(msg)
-      raise(Puppet::ParseError, "The policy is invalid - #{msg}")
-    end
+module Puppet
```

Comment:
here, i'd just add a comment that disables the check
the above case is the reason why i left the check in.

on a side-note: we should *actually* use the puppet_x pattern here, rather than effectively monkeypatching Puppet::Util…

or, in our case, puppetcommunity_x